bundles/org.eclipse.e4.ui.workbench/META-INF/MANIFEST.MF

```diff
@@ -55,3 +55,9 @@ Require-Capability: osgi.extender;
   filter:="(objectClass=org.osgi.service.event.EventAdmin)";
   effective:="active"
 Automatic-Module-Name: org.eclipse.e4.ui.workbench
+Service-Component: OSGI-INF/org.eclipse.e4.ui.internal.LocaleChangeServiceContextFunction.xml,
```

Comment:
Please undo this change, you should see warnings in IDE about duplicated headers.

Reply:
Done